### PR TITLE
Add default rustfmt

### PR DESCRIPTION
**Pull request by TornaxO7**
Adding `rustfmt.toml` will format the code on each system with the same setttings.

### CODE REVIEW

**Comment by robin-nitrokey** [comment]
Thanks for the PR!  I’ll add a target for executing rustfmt to the Makefile and then merge this PR.

File: components/usbd-ctaphid/src/class.rs [hunk 1]
```diff
@@ -124,35 +130,49 @@ const HID_REPORT_DESCRIPTOR: u8 = 0x22;
...
+    0x00, // Logical Maximum (255)
+    0x75,
+    0x08, // Report Size (8 bits)
+    0x95,
+    PACKET_SIZE as u8, // Report Count (64 fields)
+    0x91,
+    0x08, // Output (Data, Variable, Absolute)
     // EndCollection
     0xC0,
 ];
```
Comment (robin-nitrokey) [inline components/usbd-ctaphid/src/class.rs:172]: Can you please revert this change and add `#[rustfmt::skip]` so that the structure of the data is preserved and the comments stay in place?

File: components/usbd-ctaphid/src/class.rs [hunk 2]
```diff
@@ -179,13 +199,18 @@ where Bus: UsbBus
...
+            HID_DESCRIPTOR,
+            &[
+                0x11,
+                0x01,                  // bcdHID (le)
+                0x00,                  // country code: universal
+                0x01,                  // number of HID report descriptors
+                HID_REPORT_DESCRIPTOR, // 1st HID report descriptor type
+                FIDO_HID_REPORT_DESCRIPTOR_LENGTH as u8,
+                0x00, // 1st HID report descriptor length in bytes as u16-be
+            ],
```
Comment (robin-nitrokey) [inline components/usbd-ctaphid/src/class.rs:212]: Same as above.

**Review by robin-nitrokey — CHANGES_REQUESTED** [review]

File: components/usbd-ctaphid/src/class.rs (hunk 1 above)
Comment (TornaxO7) [reply components/usbd-ctaphid/src/class.rs]: done

File: components/usbd-ctaphid/src/class.rs (hunk 2 above)
Comment (TornaxO7) [reply components/usbd-ctaphid/src/class.rs]: done

**Review by robin-nitrokey — APPROVED** [review]
Thank you!
